src/js/components/ProfilePhoto.tsx

```diff
@@ -0,0 +1,81 @@
+import {Component} from 'preact';
+import Helpers from '../Helpers';
+import $ from 'jquery';
+import Session from '../Session';
+
+type Props = { photo?: string;};
+const ANIMATE_DURATION = 200;
+
+class ProfilePhoto extends Component<Props> {
+
+    render() {
+
+        return (
+          <img class="profile-photo" src={this.props.photo} onClick={e => { this.imageClicked(e); }} />
```

Comment:
SafeImg component should be used here. It checks that the src starts with `data:image`. Otherwise someone can set a remote url as img src and log the http requests to it.